### ISSUE TITLE
Different learning rate in Different layers?

### ISSUE THREAD

**Issue by meanmee** [fchollet/keras#2132]
How to set the learning rate for each layer respectively? i.e. first layer's learning rate is  0.1, last layer's learning rate is 0.001

**Comment by philipperemy** [comment]
It's coming #1991

**Comment by around1991** [comment]
Alternatively, you can use a per-parameter adaptive learning algorithm like Adadelta or Adam.
